test/lit/passes/optimize-instructions-gc.wast

```diff
@@ -3439,4 +3439,232 @@
       )
     )
   )
+
+  ;; CHECK:      (func $array.new_fixed_fallthrough (type $void)
+  ;; CHECK-NEXT:  (local $0 i32)
+  ;; CHECK-NEXT:  (local $1 i32)
+  ;; CHECK-NEXT:  (local $2 i32)
+  ;; CHECK-NEXT:  (local $3 i32)
+  ;; CHECK-NEXT:  (drop
+  ;; CHECK-NEXT:   (block (result (ref $array))
+  ;; CHECK-NEXT:    (local.set $0
```

Comment:
Why do we set this local if we never use it?

Reply:
The utility we use puts all children in locals. After that, they are all either `local.get`s or constants and we can reason about them easily. It does mean we depend on later passes to clean things up, but it save complexity (that utility isn't aware of how we will use the children).

Reply:
Makes sense, thanks.